bin/submitControlWorkflow.sh

```diff
@@ -145,9 +145,9 @@ fi
 nextflowCommand="nextflow run -N $SCXA_REPORT_EMAIL -resume $(pwd)/workflow/${workflow}/main.nf $expNamePart $skipQuantificationPart $skipAggregationPart $tertiaryWorkflowPart $skipTertiaryPart $galaxyCredentialsPart $overwritePart --enaSshUser fg_atlas_sc -work-dir $workingDir"
 echo "$nextflowCommand"
 
-# Run the LSF submission if it's not already running
+# Run the SLURM submission if it's not already running
 
-bjobs -w | grep " ${controlJobName}" > /dev/null 2>&1
+squeue -o "%.100j" | grep -w "${controlJobName}" > /dev/null 2>&1
```

Comment:
Here, the following is enough for very long job names, will avoid extra spaces in output:
```suggestion
squeue -o "%j" | grep -w "${controlJobName}" > /dev/null 2>&1
```